spec/overlap/multi_part_overlap_spec.rb

```diff
@@ -30,113 +28,123 @@
   before(:each) do
     Cluster.each(&:delete)
     c.save
-    Clustering::ClusterHtItem.new(ht_w_ec).cluster.tap(&:save)
+    Clustering::ClusterHtItem.new(htitem_with_enumchron).cluster.tap(&:save)
   end
 
   describe "#matching_holdings" do
     it "finds holdings that match on enum" do
-      cluster = Clustering::ClusterHolding.new(h_w_ec).cluster.tap(&:save)
-      overlap = described_class.new(cluster, h_w_ec.organization, ht_w_ec)
+      cluster = Clustering::ClusterHolding.new(holding_with_enumchron).cluster.tap(&:save)
+      overlap = described_class.new(cluster, holding_with_enumchron.organization, htitem_with_enumchron)
       expect(overlap.matching_holdings).to be_a(Enumerable)
       expect(overlap.matching_holdings.count).to eq(1)
     end
 
     it "finds holdings with no enum" do
-      cluster = Clustering::ClusterHolding.new(h_wo_ec).cluster.tap(&:save)
-      overlap = described_class.new(cluster, h_wo_ec.organization, ht_w_ec)
+      holding_without_enumchron = build(:holding, ocn: c.ocns.first, enum_chron: "", n_enum: "")
+      cluster = Clustering::ClusterHolding.new(holding_without_enumchron).cluster.tap(&:save)
+      overlap = described_class.new(cluster, holding_without_enumchron.organization, htitem_with_enumchron)
+      expect(overlap.matching_holdings).to be_a(Enumerable)
+      expect(overlap.matching_holdings.count).to eq(1)
+    end
+
+    it "finds holdings with nil enum" do
```

Comment:
This is the new test; the other changes here are just trying to make it a bit more clear what's going on.